### DISCUSSION TITLE
Show [@citekey] links in Obsidian's Backlinks pane

### DISCUSSION THREAD

**Discussion by mo-seph** [aidenlx/obsidian-zotero#514] [Ideas]
Because ZotLit does something clever with link processing, the [@citekey] format links to literature notes are not treated as Obsidian links, so they don't show up in the Backlinks pane. It would be great if there was a way to see all the Notes that link to a particular Literature Note

**Comment by aidenlx** [comment]
Will be implemented: https://github.com/aidenlx/zotlit/issues/689

**Comment by aidenlx** [comment]
implemented in [v2.1.0-beta.1](https://zotlit.aidenlx.site/changelog/2.1.0-beta.1)
